core/templates/dev/head/editor/ExplorationFeedback.js

```diff
@@ -161,23 +236,48 @@ oppia.controller('ExplorationFeedback', [
     {id: 'not_actionable', text: 'Not Actionable'}
   ];
 
-  $scope.getHumanReadableStatus = function(status) {
-    for (var i = 0; i < $scope.STATUS_CHOICES.length; i++) {
-      if ($scope.STATUS_CHOICES[i].id === status) {
-        return $scope.STATUS_CHOICES[i].text;
+  return {
+    STATUS_CHOICES: angular.copy(_STATUS_CHOICES),
+    getLabelClass: function(status) {
```

Comment:
Hm, I thought about it, and your arguments are reasonable. I decided I didn't care too much either way, and went to change it.

Then I found out that we do this in two files -- the directive and the main HTML -- and that swung me the other way, since it's a complicated enough construction that I'd prefer to do it in a single place (DRY), so we run less risk of someone updating one of the styling tags but not the other. It doesn't seem worth factoring that bit out into a separate directive, so overall I think I'm still inclined to leave it as-is.